src/runtime/TypeManager.cs

```diff
@@ -475,17 +475,20 @@ internal static PyType CreateMetatypeWithGCHandleOffset()
             int size = Util.ReadInt32(Runtime.PyTypeType, TypeOffset.tp_basicsize)
                        + IntPtr.Size // tp_clr_inst_offset
             ;
-            var result = new PyType(new TypeSpec("clr._internal.GCOffsetBase", basicSize: size,
-                new TypeSpec.Slot[]
-                {
-
-                },
-                TypeFlags.Default | TypeFlags.HeapType | TypeFlags.HaveGC),
-                bases: new PyTuple(new[] { py_type }));
 
-            SetRequiredSlots(result, seen: new HashSet<string>());
-
-            Runtime.PyType_Modified(result);
+            var slots = new[] {
+                new TypeSpec.Slot(TypeSlotID.tp_traverse, subtype_traverse),
+                new TypeSpec.Slot(TypeSlotID.tp_clear, subtype_clear)
+            };
```

Comment:
This creates multiple sources of truth

Reply:
It doesn't. Python 3.11 rejects creating a type that has `HAVE_GC` set but not these slots. Rightly so :)

Reply:
Can we have this array shared with the other place that used to use this function?